dbms/src/Storages/Page/V3/PageDirectoryFactory.cpp

```diff
@@ -135,6 +135,11 @@ bool PageDirectoryFactory::applyRecord(
         iter->second = std::make_shared<VersionedPageEntries>();
     }
 
+    if (r.page_id.low > dir->max_page_id)
```

Comment:
How about we add some trace logs here? It seems that there are frequent bugs here.

Reply:
This will result in too many logs.

Reply:
added log after mvcc restored